src/api/lib/activexml/node.rb

```diff
@@ -40,9 +40,6 @@ def reset_runtime
         @@xml_time = 0
       end
 
-      # transport object, gets defined according to configuration when Base is subclassed
-      attr_reader :transport
```

Comment:
are you sure that you can remove this? I know that there is also a `transport` method in this class, but if I am not wrong, without this you can not access the `@@transport` variable from outside anymore. And as it not used here neither... why do we have the `@@transport` object variable? 🤔